pages/index.js

```diff
@@ -1,3 +1,27 @@
+import styled from "styled-components";
+import FoodCard from "../components/FoodCard";
+import useSWR from "swr";
+import { fetcher } from "../helpers/api";
+
 export default function Home() {
-  return <h1>Hello Flamingos! 🦩</h1>;
+  const { data: data, error } = useSWR("/api/recipes", fetcher);
+
+  if (error) return <h1>There was an error</h1>;
+
+  return (
+    <>
```

Comment:
We always want to have a `main` element, so I would use that one here instead.
```suggestion
    <main>
```